src/GraphQl/Resolver/Factory/ResolverFactory.php

```diff
@@ -24,20 +25,23 @@ class ResolverFactory implements ResolverFactoryInterface
 {
     public function __construct(
         private readonly ProviderInterface $provider,
-        private readonly ProcessorInterface $processor
+        private readonly ProcessorInterface $processor,
+        private readonly PropertyMetadataFactoryInterface $propertyMetadataFactory
```

Comment:
this needs to be nullable for backward compatibility

Reply:
Done